build-system/babel-plugins/babel-plugin-transform-amp-asserts/index.js

```diff
@@ -74,7 +74,10 @@ module.exports = function (babel) {
 
       // If we can statically evaluate the value to a falsey expression
       if (evaluation.confident) {
-        if (type) {
+        if (evaluation.value === null) {
+          path.replaceWith(t.nullLiteral());
+          return;
```

Comment:
This is all the custom pass was handling?

Reply:
No, @jridgewell wrote this transform and the entire contents were previously handled by Closure.

The result here still isn’t a 1:1 match with beforehand. Work remains.